LoopKitUI/Extensions/CGFloat.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  CGFloat.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2020-01-27.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+extension CGFloat {
```

Comment:
[nit] 

<rant>
This is not a complaint against this specific code, but rather against this common Swift practice of extending a type to add properties like this. What do any of these types have to do with `CGFloat` other than they happen to be `CGFloat`s? IMHO, this is an anti-pattern and just pollutes the type's namespace. To me, it would make far more sense to have a new class called `LoopUICharacteristics` (or something like that since I suck at naming things) and collect up all of the related properties in that class. Yes, I get that it *looks* clever when you can do `.loopCollapsedSectionHeaderHeight` as a parameter (since it is an extension of `CGFloat`) rather than a fully qualified `LoopUICharacteristics.collapsedSectionHeaderHeight`, but IMO it is just misleading. With this route, you'd potentially need to extend a large number of types (eg. `CGFloat`, `UIColor`, `UIFont`, etc.) depending upon what you're trying to configure and they'd be spread all over the place. 

Sorry, nothing personal, but this practice just drives me bananas. Yes, I've done it myself to be consistent, but it makes no sense to me. ;)
</rant>